### ISSUE TITLE
Upload profile image not showing correctly

### ISSUE THREAD

**Issue by capiscuas** [FreedomCoop/valuenetwork#170]
Hi,
after my first profile login, when I uploaded my profile photo this is what appears:

![image](https://cloud.githubusercontent.com/assets/515154/19106752/5f2923ca-8ae9-11e6-9311-f1ece8113bbb.png)

The address of that "broken" image is the following
https://ocp.freedomcoop.eu/site_media/media/photos/profile.jpg.128x128_q85.jpg

so I don't know what happened with the one that I uploaded.
Thanks.

**Comment by bhaugen** [comment]
@capiscuas thanks for reporting the problem, I'll check it out.

I also notice your suggestion of adding Python/Django skills. Do you have such skills?

**Comment by bhaugen** [comment]
@XaviP do you think we might have a directory permissions problem? Works fine in testocp, same code.

**Comment by capiscuas** [comment]
@bhaugen Yes I do. Check my open source repositories for python and Django code:
github.com/capiscuas/

**Comment by bhaugen** [comment]
That's wonderful! Want to do some coding on OCP?
https://github.com/FreedomCoop/valuenetwork

**Comment by capiscuas** [comment]
Dankon. I will take a look at the project. Looks very interesting. Although right now I am in my quest for a remunerated job as a programmer, so I may not have much time.

**Comment by bhaugen** [comment]
No worries. We understand not much time...
